make_composer.mk

```diff
@@ -37,8 +37,9 @@ docker-e2e-playwright: ## Create a local docker deployment.
 	-v $$PWD/ui:/app/ui \
 	-e NODE_TLS_REJECT_UNAUTHORIZED=0 \
     -e PLAYWRIGHT_USE_VITE=FALSE \
-    -e PLAYWRIGHT_UI_BASE_URL=http://127.0.0.1:80 \
+    -e PLAYWRIGHT_UI_BASE_URL=http://127.0.0.1 \
     -e PLAYWRITE_SHARD=1/1 \
+    -e PLAYWRITE_CI=TRUE \
```

Comment:
This seems to be a typo, everywhere else we use the spelling "PLAYWRIGHT"?

I would strongly urge us to use the correct spelling everywhere, even if this requires changing backend code / env variables not covered in this PR, so that it can easily be found later. Probably @sir-hassan could reply to this?

Reply:
@sir-hassan I will try to fix this and come back to you if I need any help. I think it's probably find and replace

Reply:
@sebxian @stefan-kracht , Oh ja thats clearly a typo. Yes its a simple find and replace fix.